source/style-guide/style/text-formatting.txt

```diff
@@ -514,12 +507,12 @@ elements. The following style differences are highlighted:
                :copyable: false
 
                The following example shows a basic configuration for
-               the FTP service, in a file in the ``/etc/xinetd.d
-               directory``.
+               the FTP service, in a file in the ``/etc/xinetd.d/``
```

Comment:
Question:

Should the double backticks exist here? If I remember correctly, the double-backtick within a code block used to have some rendering behavior associated with it, but I think that was removed a while back. If uncertain or if there's some guidance around representing monospace paths this way, no changes necessary!

Reply:
I'm not aware of any prohibitions, but the section renders properly. Leaving as is for now.